test/com/google/enterprise/adaptor/prebuilt/PrebuiltTransformsTest.java

```diff
@@ -706,20 +708,21 @@ public void testDeleteToString() {
 
   @Test
   public void testDeleteToStringWithKeys() {
-    Map<String, String> config = new HashMap<String, String>();
+    Map<String, String> config = new LinkedHashMap<String, String>();
     config.put("key2", "colour");
     config.put("key3", "author");
     config.put("keyset3", "params");
     config = Collections.unmodifiableMap(config);
 
     MetadataTransform transform = PrebuiltTransforms.deleteMetadata(config);
-    assertEquals("DeleteTransform(keys=[(key=colour,keyset=metadata), "
-        + "(key=author,keyset=params)])", transform.toString());
+    assertEquals("DeleteTransform("
```

Comment:
Minor preference for having the two pieces that need to be in order on the same line, in order.  As dumb as it is I looked multiple times before i realized the actual was out of order from expected. I think having on one line would have helped me a small amount.  It seem that this part of the string is the key part to see.